options.go

```diff
@@ -1254,6 +1254,15 @@ type NearblackOption interface {
 	setNearblackOpt(nbOpt *nearBlackOpts)
 }
 
+type setGCPsOpts struct {
+	errorHandler ErrorHandler
```

Comment:
```suggestion
	errorHandler ErrorHandler
	projString string
	sr *Spatialref
```
c.f. SetGCPs and SetGCPs2 unification comment